public/styles/components/_footer.scss

```diff
@@ -0,0 +1,161 @@
+.bsds-footer {
+  @include bsds-defaults-type;
+  background-color: $neutral-95;
+
+  .bsds-nav-footer {
+    @include font-size($fs-body-subtle);
+  }
+
+  .bsds-nav-group {
+    &:not(:first-of-type) {
+      margin-top: space(6x);
+    }
+  }
+
+  .bsds-nav-title {
+    @include font-family-base-heavy;
+
+    + .bsds-nav {
+      margin-top: space(lg);
+    }
+  }
+
+  .bsds-nav {
+    margin: 0;
+    padding: 0;
+  }
+
+  .bsds-nav-item {
+    list-style: none;
+  }
+
+  .bsds-link-nav {
+    @include link-navigation;
+  }
+
+  .bsds-footer-base {
+    padding-bottom: space(6x);
+
+    .bsds-footer-base-container {
+      @include font-size($fs-body-subtle);
+      display: flex;
+      flex-direction: column;
+      justify-content: flex-start;
+      align-items: flex-start;
+      padding: 0 space(3x);
+
+      .bsds-footer-logo {
+        width: auto;
+        max-height: space(8x);
+        margin-top: space(3x);
+      }
+
+      .bsds-footer-text-small {
+        @include font-size($fs-small);
+        margin-top: space(lg);
+      }
+
+      .bsds-footer-link-corp {
+        @include link-navigation;
+        @include font-size($fs-small);
+        margin-top: space(lg);
```

Comment:
we could also create a file-level mixin for these two styles and use that in abstracted and custom classes to apply throughout the file